modules/initrd-boot-gui.nix

```diff
@@ -18,27 +20,37 @@ let
   '';
 in
 {
-  mobile.boot.stage-1.contents = with pkgs; [
-    {
-      object = "${pkgs.mobile-nixos.stage-1.boot-error}/libexec/boot-error.mrb";
-      symlink = "/applets/boot-error.mrb";
-    }
-    {
-      object = "${pkgs.mobile-nixos.stage-1.boot-splash}/libexec/boot-splash.mrb";
-      symlink = "/applets/boot-splash.mrb";
-    }
-    {
-      object = "${pkgs.mobile-nixos.stage-1.boot-recovery-menu}/libexec/boot-recovery-menu.mrb";
-      symlink = "/applets/boot-selection.mrb";
-    }
-    {
-      object = "${minimalX11Config}";
-      symlink = "/etc/X11";
-    }
-  ];
+  options.mobile.boot.stage-1.gui = {
+    enable = mkOption {
+      type = types.bool;
+      default = true;
+      description = "enable splash and boot selection GUI";
```

Comment:
Can we add a note explaining that this will also disable the passphrase input? And also maybe a note that disabling this does not necessarily means that the fbcon consoles will show.

Reply:
Better yet: assert! (I'm doing it.)